azure-pipelines-integration.yml

```diff
@@ -3,12 +3,8 @@ trigger:
 - master
 - master-vs-deps
 - release/*
-- dev16.0
```

Comment:
Did we move the dev16.0 branches under release/ ?

Reply:
No but this won't affect them. This change doesn't flow to dev16, code flows the other direction. Hence this won't affect them.